src/main/java/WebRTCConf.java

```diff
@@ -0,0 +1,244 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.safari.SafariOptions;
+
+
+
+public class WebRTCConf {
+   
+   public static int getRoomID() {
+      Random random = new Random();
+        int roomId = random.nextInt(500000);
+        roomId += 100000;
+        
+        return roomId;
+   }
+   
+   public static ChromeOptions getFireFoxConfiguration() {
```

Comment:
check if ChromeOptions has a parent class or can we specify correct options class as return type for each browser?

Reply:
ChromeOptions has MutableCapabilities as parent class. Modified the code accordingly.